src/App.js

```diff
@@ -1,19 +1,33 @@
 import './App.css';
-import initialCards from './cards-data';
+// import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+// import { useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { useGameContext } from './context/GameContext.js';
 
 function App() {
-  const [deck, setDeck] = useState(initialCards);
-  const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
-  const [playerTwoHand, setPlayerTwoHand] = useState([]);
-  const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
-
+  const {
+    deck,
+    selectedCard,
+    playerOneHand,
+    playerTwoHand,
+    playerThreeHand,
+    setPlayerOneHand,
+    setPlayerTwoHand,
+    setPlayerThreeHand,
+    to,
+    from,
+    setDeck,
+    setSelectedCard,
+  } = useGameContext();
+  // const [deck, setDeck] = useState(initialCards);
+  // const [playerOneHand, setPlayerOneHand] = useState([]);
+  // const [selectedCard, setSelectedCard] = useState();
+  // const [playerTwoHand, setPlayerTwoHand] = useState([]);
+  // const [playerThreeHand, setPlayerThreeHand] = useState([]);
+  // const [from, setFrom] = useState('deck');
+  // const [to, setTo] = useState(1);
   function findCardIndex(value, suit, cards) {
```

Comment:
Notice how these functions are not being used here? The only reason they were living in `app` instead of `ExecutePassFunction` was because of state limitations that have been fixed by putting the state in context. Try to see if you can move them out of here and into where they are being used. One less prop to pass, a cleaner-looking `app.js`.